packages/cli-core/src/format/format.ts

```diff
@@ -0,0 +1,19 @@
+import * as os from 'node:os';
+
+/**
+ * Provides formatting utilities.
+ */
+export const format = {
+  list: (lines: string[]) =>
+    lines.map((line: string) => ` - ${line}`).join(os.EOL),
+  indent: (message: string, indentLevel: number) => {
```

Comment:
We can omit `indentLevel` and just always indent by 2 spaces. If you want to indent more, you chain calls to indent together.